src/lib/jsvat.js

```diff
@@ -0,0 +1,70 @@
+readonly[key];
```

Comment:
This looks like some bogus file.
Thanks for PR btw!

Reply:
Yes, definitely a file i added on accident 👍  removed it